src/main/java/com/WebController/EventController.java

```diff
@@ -63,4 +81,53 @@ public String receivePatient (@RequestBody List<Event> events){
         return list;
     }
 
+    @PostMapping(value = "/userregister", consumes = "application/json")
+    @ResponseBody
+    public String userRegistration(@RequestBody User user){
+        if (this.userRepository.findUserByEmailAddressIgnoreCase(user.getEmailAddress())!=null){
+            return "user already exists";
+        }
+        userRepository.save(user);
+        ConfirmationToken token = new ConfirmationToken(user);
+
+        SimpleMailMessage message = new SimpleMailMessage();
+        message.setTo(user.getEmailAddress());
+        message.setSubject("COVID-19 TRacking System Registration.");
+        message.setFrom("covidtracking08@gmail.com");
```

Comment:
Magic email address! Best to store this as a Resource somewhere, and retrieve it as a property.